rules_128tech/pylint_plugins/BUILD.bazel

```diff
@@ -0,0 +1,18 @@
+load("@rules_python//python:defs.bzl", "py_library")
+
+package(default_visibility = ["//visibility:public"])
+
+py_library(
+    name = "mock_patch_autospec",
+    testonly = True,
+    srcs = [
+        ":__init__.py",
+        ":mock_patch_autospec.py",
+    ],
+    imports = ["."],
```

Comment:
I wonder if this import will mess up downstream libraries paths... in the case of the pytest plugin I wrote I needed to specify `-p rules_128tech.pytest_plugins.pytest_bazel_sharder`, and the `pytest_bazel_sharder` library target had no `imports` directive.

Maybe it won't be an issue for pylint because it doesn't _execute_ the modules it's linting, but just an FYI

Reply:
Good point, this was the fastest way to get it working but I can probably do something similar by fully qualifying `rules_128tech.pylint_plugins.mock_patch_autospec` and remove this.